ui/app/AppLayouts/Profile/helpers/ProfileShowcaseDirtyState.qml

```diff
@@ -48,8 +48,8 @@ QObject {
         writable.setVisibility(key, visibility)
     }
 
-    function changePosition(key, to) {
-        writable.changePosition(key, to)
```

Comment:
Don't we need as well to orchestrate the change position from the _writable_ model?